WeatherStation/v2/_summit.yaml

```diff
@@ -0,0 +1,8 @@
+instances:
+  - sal_index: 1
+    ws_type: lsst
+    config:
+      host: '139.229.170.220'
+      port: 50000
+      buffer_size: 2048
+      timeout: 120.0
```

Comment:
I strongly suggest only specifying host here to avoid repetition with `_init.yaml` and the illusion that you are setting site-specific values for those parameters.

Reply:
I removed the comment. And I removed the superfluous parameters. I am unsure about the exact format though. Can I **only** specify the host or do I need to keep the structure like this:

```
instances:
  - sal_index: 1
    ws_type: lsst
    config:
      host: 'the_host_name'
```

Reply:
It needs to look like your example, with all the intermediate levels.